ClientSideKeyVaultKeyToCustomerManagedKey/config.py

```diff
@@ -0,0 +1,27 @@
+from azure.keyvault.keys.crypto import KeyWrapAlgorithm
+# the below values are fake and need to be replaced
+
+KEYVAULT_URL = ""
+CLIENT_ID = ""
+CLIENT_SECRET = ""
+TENANT_ID = ""
+connection_str = ""
+
+# items to make encryption scope
```

Comment:
These are storage account info, SUB_ID is where that storage account under. So I think we should move connection_str here instead of group it together with Keyvault info.
Also can we make these names in this file all upper case?